samza-yarn/src/main/scala/org/apache/samza/job/yarn/YarnJobFactory.scala

```diff
@@ -37,6 +38,12 @@ class YarnJobFactory extends StreamJobFactory {
     if (config.containsKey(YarnConfiguration.RM_ADDRESS)) {
       hConfig.set(YarnConfiguration.RM_ADDRESS, config.get(YarnConfiguration.RM_ADDRESS, "0.0.0.0:8032"))
     }
+
+    if (config.containsKey("fs.certfs.impl.override")) { //TODO: change to use constants from CertFSConstants once that CertFSConstants.java is in
+      hConfig.set("fs.certfs.impl", config.get("fs.certfs.impl.override"))
+      logger.info("samza job config fs.certfs.impl.override is used for yarn.")
```

Comment:
changed to use fs.<scheme>.impl and made the same config name for both samza job config and yarn config.